tests/ipv6-non-temp-addr-tests.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+
+# Copyright (C) 2018 Aravind Prasad S
+#
+# License: BSD
+
+srcdir="${srcdir:-.}"
+
+echo "***********************************************"
+echo "This test will use a radius server on localhost"
+echo "and which can be executed with run-server.sh   "
+echo "***********************************************"
+
+TMPFILE=tmp$$.out
+
+#Check if its linux system
+linux_env_var="$(uname -s)"                                                                               
+if test  $env_var != "Linux";then                                                                   
+    echo "Not linux. Tests cannot be done"                                                                                
+    exit 0                                                                                          
+fi
+
+if test -z "$SERVER_IP";then
+	echo "the variable SERVER_IP is not defined"
+	exit 77
+fi
+
+PID=$$
+sed -e 's/::1/'$SERVER_IP6'/g' -e 's/servers-ipv6-temp/servers-ipv6-temp'$PID'/g' <$srcdir/radiusclient-ipv6.conf >radiusclient-temp$PID.conf
+sed 's/::1/'$SERVER_IP6'/g' <$srcdir/servers-ipv6 >servers-ipv6-temp$PID                            
+echo "ipv6_use_non_temp_addr	true" >> radiusclient-temp$PID.conf
+
+#enable ipv6 privay settings
+sudo sysctl -w net.ipv6.conf.all.use_tempaddr=2
```

Comment:
I do not think we need to use sudo in the script; travis already runs  as root, and it is not a good practice to call sudo in a test suite. We can use:
```
        if test "$(id -u)" != "0";then
                exit 77
        fi
```
in the beginning of the script to ensure that this will be skipped if the testsuite is run as root.